code/style.css

```diff
@@ -1,4 +1,232 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+/*** HEADER ***/
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+  margin-top: 20px;
 }
+
+.logo {
+  width: 20%;
+  margin-left: 10px
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-el {
+  display: none;
+}
+
+/*** HAMBURGER MENU ***/
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 20px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 25px;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 2px;
+  background-color: gray;
+  border-radius: 5px;
+}
+
+
+
+h1, h2 {
+  font-family: 'Montserrat', sans-serif;
+  text-transform: uppercase;
+}
+
+footer {
+  text-align: center;
+  background-color: #DDDDDD;
+  height: 70px;
+  margin-top: 60px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+footer p {
+  padding: 20px auto;
+  font-family: 'Montserrat', sans-serif;
+  color: #A4A4A4;
+  font-size: 10px;
+}
+
+/*** MAIN NEWS ***/
+
+.big-news {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-bottom: 30px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.big-news img {
+  width: 100%;
+  margin: 0;
+}
+
+.big-news h1 {
+  margin: 8px 10px 0px;
+  font-weight: 600;
+}
+
+.big-news p {
+  margin: 5px 10px 12px;
+}
+
+.btn {
+  display: none;
+}
+
+.dots {
+  border: dotted #A4A4A4 6px;
+  border-bottom: none;
+  width: 4%;
+  margin: 30px auto;
+}
+
+/*** NEWS CARDS***/
+
+.news-card-section {
+  display: flex;
+  flex-direction: column;
+  margin-right: 10%;
+  margin-left: 10%;
+}
+
+.news-card {
+  margin-bottom: 30px;
+  box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.2), 0 3px 10px 0 rgba(0, 0, 0, 0.19);
+  opacity: 1;
+}
+
+.news-card:hover {
+  opacity: 0.5;
+}
+
+.news-card img {
+  width: 100%;
+}
+
+.news-card h2 {
+  margin-top: 5px;
+  padding: 0px 15px;
+  font-weight: 500;
+}
+
+
+/*** TABLET ***/
+
+@media (min-width: 768px) {
+  
+  .hamburger {
+    display: none;
+  }
+
+  .nav {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+  .nav-elements {
+    min-width: 442px;
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 10px;
+  }
+
+  .nav-el {
+    display: inline;
+    padding: 10px 5px 10px;
+    text-decoration: none;
+    color: black;
+    margin-right: 5px;
+  }
+  .nav-el:nth-child(4) {
+    margin-right: 0px;
+  }
+
+  a:hover {
+    background-color: white; 
+    color: black; 
+    border: 0.5px solid #A4A4A4;
+  }
```

Comment:
Nice hover effect! Do you notice that the elements "jump" a little bit on hover though? It's because the element suddenly takes up more space now that it has a border. A little hack you could use here is to have a transparent border from the start, and then just change the color on hover :)